editor/src/clj/editor/changes_view.clj

```diff
@@ -118,7 +118,7 @@
                         :result true}]})
       (let [moved-files (mapv #(vector (path->file workspace (:new-path %)) (path->file workspace (:old-path %))) (filter #(= (:change-type %) :rename) selection))]
         (git/revert git (mapv (fn [status] (or (:new-path status) (:old-path status))) selection))
-        (async-reload! workspace changes-view moved-files)))))
+        (async-reload! changes-view moved-files)))))
```

Comment:
Changed `async-reload!` callback signature to only provide the `changes-view` and the `moved-files`.